test/integration/sds_dynamic_integration_test.cc

```diff
@@ -408,6 +475,18 @@ class SdsDynamicDownstreamCertValidationContextTest : public SdsDynamicDownstrea
     client_ssl_ctx_ = createClientSslTransportSocketFactory({}, context_manager_, *api_);
   }
 
+  void configInlinedCerts(
+      envoy::extensions::transport_sockets::tls::v3::CommonTlsContext* common_tls_context) {
+    common_tls_context->add_alpn_protocols(Http::Utility::AlpnNames::get().Http11);
```

Comment:
Maybe pull this out or rename function to make it clear that we're also setting the ALPN?

Reply:
moved it out.